includes/rest/class-inbox-controller.php

```diff
@@ -368,8 +369,12 @@ public function get_item_schema() {
 	 * @return array An array of user IDs who are the recipients of the activity.
 	 */
 	private function get_local_recipients( $activity ) {
-		if ( is_activity_public( $activity ) ) { // Public activity, deliver to followers of the actor.
-			return Following::get_follower_ids( $activity['actor'] );
+		if ( is_activity_public( $activity ) ) {
+			if ( is_activity_reply( $activity ) ) {
+				return Actors::get_all_ids();
+			} else {
+				return Following::get_follower_ids( $activity['actor'] );
+			}
```

Comment:
```suggestion
		$user_ids = array();
		if ( is_activity_public( $activity ) ) {
			$user_ids = Following::get_follower_ids( $activity['actor'] );
		}
```
And then removing the `$user_ids   = array();` in line 383. I'm curious if that works?

Then we'd default to followers and add any specific account mentioned in replies.

Reply:
oh, done in 50cd03d.